UNRELEASED.md

```diff
@@ -1,4 +1,8 @@
 # Unreleased
 
 _Nothing yet..._
```

Comment:
Can we remove this line (and never add it again)?

Reply:
I'll clean it up when I release